src/bin/rendezvous_server.rs

```diff
@@ -49,8 +52,31 @@ async fn main() -> Result<()> {
 
         if let Some(event) = event {
             match event {
-                SwarmEvent::Behaviour(Event::Ping(_)) => {}
-                event => println!("swarm event: {:?}", event),
+                SwarmEvent::Behaviour(Event::Rendezvous(RendezvousEvent::PeerRegistered {
+                    peer,
+                    registration,
+                })) => {
+                    tracing::info!(%peer, namespace=%registration.namespace, addresses=?registration.record.addresses(), ttl=%registration.ttl,  "Peer Registered");
```

Comment:
Nit:

```suggestion
                    tracing::info!(%peer, namespace=%registration.namespace, addresses=?registration.record.addresses(), ttl=%registration.ttl, "Peer registered");
```

Reply:
Also, any reason why you went for `%` again for the ttl?

Reply:
nope, must of lost it when stashing etc